packages/packages/api/src/medical/models/document.ts

```diff
@@ -34,9 +34,16 @@ export const documentQueryProgress = z.object({
 
 export type DocumentQueryProgress = z.infer<typeof documentQueryProgress>;
 
+export const progressSchema = z.object({
+  status: documentQueryStatusSchema,
+  total: z.number().optional(),
+  successful: z.number().optional(),
+  errors: z.number().optional(),
+});
+
 export const documentQuerySchema = z.object({
-  queryStatus: documentQueryStatusSchema,
-  queryProgress: documentQueryProgress.optional(),
```

Comment:
Could we keep these as deprecated for a few weeks? In case we can map the new structure to those somehow... just thinking to minimize disruption to customers relying on this.